website/pages/blog/postgres-cdc-to-any-destination.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: PostgreSQL CDC (Change-Data-Capture) to any destination
+tag: announcement
+date: 2023/12/14
+description: >-
+  In this blog we will talk about how to stream changes from PostgreSQL
+  to any destination with CloudQuery and no queue or Kafka required.
+author: yevgenypats
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+One of the most powerful features of PostgreSQL is its ability to replicate data, and in this article, we'll take a deep dive into the topic of logical replication.
+
```

Comment:
Maybe we can add a section here to show how CloudQuery now allows you to sync from postgres, before diving into the internals?